schemas/bedrock-kms-http.js

```diff
@@ -0,0 +1,139 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const controller = {
+  title: 'controller',
+  type: 'string'
+};
+
+const postKeystore = {
+  title: 'postKeystore',
+  type: 'object',
+  additionalProperties: true,
+  required: ['sequence', 'controller'],
+  properties: {controller,
+    sequence: {
```

Comment:
we should be able to say min/max values for this.

Reply:
Added.